src/calendar/CalendarMonth.jsx

```diff
@@ -76,10 +77,15 @@ const CalendarMonth = React.createClass({
 
   renderDayHeaders() {
     let {firstOfWeek} = this.props;
-    let indices = Immutable.Range(firstOfWeek, 7).concat(Immutable.Range(0, firstOfWeek));
+    let indices = Immutable.Range(firstOfWeek, 7).concat(Immutable.Range(0, Math.min(7, firstOfWeek)));
```

Comment:
Hi, sorry it took me a few days to get back to you for such simple answers.
Absolutely right, this is not required.